main.c

```diff
@@ -392,12 +393,15 @@ void update_info(void)
 	}
 }
 
-int ptr_third_x(void)
+int nav_button(void)
 {
-	int x, y;
+	int x, y, nw;
 
 	win_cursor_pos(&win, &x, &y);
-	return MAX(0, MIN(2, (x / (win.w * 0.33))));
+	nw = (NAV_WIDTH <= 0 ? 0.33 : NAV_WIDTH) * (NAV_WIDTH >= 1 ? 1 : win.w);
+	nw = MIN(nw, win.w / 2);
+
+	return x <= nw ? 0 : x < win.w - nw ? 1 : 2;
```

Comment:
Use a better variable name than `nw`. navigation_width?

Reply:
I was going with suckless style (with doubt myself), maybe `nav_width` or something will be better (not that long)?

Reply:
I think nav_width works